webapp/src/components/sidebar_buttons/index.js

```diff
@@ -26,6 +25,8 @@ function mapDispatchToProps(dispatch) {
             getUnreads,
             getYourPrs,
             getYourAssignments,
+            updateRhsState,
+            dispatch: (action) => dispatch(action),
```

Comment:
Seems a bit strange to pass though the dispatch. (see other comment)

Reply:
Agreed